### DISCUSSION TITLE
Query to show count

### DISCUSSION THREAD

**Discussion by ChrisWeAspire** [Budibase/budibase#9127] [Help]
Hi,

New to budibase, I have a table of data that includes a time stamp.

I want to return the count of records that have the current date as the time stamp.

I have added the source, added the data provider object then added the stat card to this. I can see you set the value to query values.

I am trying to work out how to create the query and unfamiliar with the syntax to show the total number of records where the timestamp field matches the current date.

Is there a way to do this via handlebar query or would it require a JS query in the other tab?

Thanks.

CW

**Comment by melohagan** [comment]
Hey @ChrisWeAspire 

Is this for the internal Budibase DB or an external DB?

**Reply by ChrisWeAspire** [reply]
Hi,

This is an external MySQL database,

Thanks.

Regards,

**Comment by melohagan** [comment]
Hey @ChrisWeAspire 

You may need to use a little bit of JavaScript for this one. I was able to get current date filtering for a MySQL Date/Time field with the following filters:

![Screenshot 2023-01-03 at 11 33 49](https://user-images.githubusercontent.com/101575380/210349458-81ce3724-26e2-431b-826d-754faea036b8.png)

JS Binding:
```js
let endOfDay = new Date();
endOfDay.setHours(23, 59, 59);
return endOfDay;
```
